packages/epics/src/people/components/person-head.tsx

```diff
@@ -91,10 +92,12 @@ export const PersonHead = ({
           </Button>
         </Skeleton>
         <Skeleton loading={isLoading} width={120} height={35}>
-          <Button colorVariant="accent">
-            <RxPencil2 />
-            Edit profile
-          </Button>
+          <Link href={`/profile/edit`} scroll={false}>
+            <Button colorVariant="accent">
+              <RxPencil2 />
+              Edit profile
+            </Button>
+          </Link>
         </Skeleton>
```

Comment:
Please compare the behaviour when `tabbing` through the focus. With `Link` wrapped around `Button`, we have two focus states, one for the Link, one for the Button.
```suggestion
          <Button asChild colorVariant="accent">
            <Link href={`/profile/edit`} scroll={false}>
              <RxPencil2 />
              Edit profile
            </Link>
          </Button>
```

Reply:
> Why have you resolved without addressing it?

regarding this, previously we always wrapped the Button in Link, why is it not correct now 🧐? In the whole project we use exactly this approach and the side-panel works

Reply:
> Please compare the behaviour when tabbing through the focus. With Link wrapped around Button, we have two focus states, one for the Link, one for the Button.

This is why 👆

Reply:
Please also check the docs on [shadcn/ui](https://ui.shadcn.com/docs/components/button) in the Link section